packages/js/product-editor/src/components/attribute-control/utils.ts

```diff
@@ -48,3 +53,36 @@ export function reorderSortableProductAttributePositions(
 		}
 	);
 }
+
+/**
+ * Checks if the given attribute has
+ * either terms (global attributes) or options (local attributes).
+ *
+ * @param {EnhancedProductAttribute} attribute - The attribute to check.
+ * @return {boolean} True if the attribute has terms or options, false otherwise.
+ */
+export const hasTermsOrOptions = (
+	attribute: EnhancedProductAttribute
```

Comment:
Should you allow for `null` here, since you are handling it? Seems like it should be handled the same way in both this function and `isAttributeFilledOut`.

Reply:
I didn't want to change the current implementations of the functions. Probably, yes.